lib/Conversation/index.js

```diff
@@ -49,7 +50,7 @@ class Conversation extends Component {
   }
 
   render() {
-    const conversationClasses = (this.state.isActive) ? 'conversation--is-active' : '';
+    const conversationClasses = (this.state.isActive || this.props.isActive) ? 'conversation--is-active' : '';
```

Comment:
A slight BEM concern; I'd argue that things like `is-active`, `is-collapsed` and whatnot are part of BEM state, not modifiers. This makes it a bit weird having in the SCSS nested classes inside modifiers.

For example, we have now:

```
.conversation--is-active {
  border-color: $conversation-border-color;

  .conversation__resolve {
    border-bottom: $conversation-border-width solid $conversation-border-color;
  }
```

It's a bit awkward deleting so much parent-like reponsibility to a modifier, instead of a state. Does this make sense? :)